openapi/quickbooks.online/Module.md

```diff
@@ -8,4 +8,48 @@ The QuickBooks Online Accounting API utilizes the REST architecture. It lets you
 Before using this connector in your Ballerina application, complete the following:
 
 * Create a [QuickBooks account](https://quickbooks.intuit.com/global/).
-* Obtain tokens by following [this guide](https://developer.intuit.com/app/developer/qbo/docs/get-started/start-developing-your-app).
+* Obtain credentials by following [this guide](https://developer.intuit.com/app/developer/qbo/docs/get-started/start-developing-your-app). Tokens can be obtained by using [OAuth 2.0 Playground](https://developer.intuit.com/app/developer/playground).
+
+## Quickstart
+
+To use the QuickBooks Online connector in your Ballerina application, update the .bal file as follows:
+
+### Step 1: Import connector
+Import the `ballerinax/quickbooks.online` module into the Ballerina project.
+```ballerina
+    import ballerinax/quickbooks.online as quickbooks;
+```
+
+### Step 2: Create a new connector instance
+Add the configuration file by creating a `Config.toml` file. Config file should have following configurations. Add the tokens obtained in the previous step to the `Config.toml` file.
```

Comment:
Updated the doc. 
Can we have it as below rather than adding as steps?
Create configuration file as `Config.toml` file. Add the tokens obtained through OAuth 2.0 Playground to the `Config.toml` file as follows.
```ballerina
    [authConfig]
    clientId: <CLIENT_ID>,
    clientSecret: <CLIENT_SECRET>,
    refreshToken: <REFRESH_TOKEN>
```

Reply:
How about the following?

```suggestion
Create a configuration file named `Config.toml` and add the tokens obtained through the OAuth 2.0 Playground to it as follows:
```

Reply:
This would be better. I will update as follows